dracut/30ignition/ignition-kargs-helper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Send out a warning message to contact the distribution and exit.
+# Distributions implementing Ignition kargs can either disable the unit
```

Comment:
They'd disable the unit if they're not implementing kargs, right?

Reply:
Yeah, switched up the wording to `Distributions implementing Ignition kargs can override this binary with the actual implementation.`